singularity-eos/eos/eos_eospac.hpp

```diff
@@ -140,7 +140,18 @@ class EOSPAC : public EosBase<EOSPAC> {
                 bool apply_smoothing = false,
                 eospacSplit apply_splitting = eospacSplit::none,
                 bool linear_interp = false);
+  PORTABLE_INLINE_FUNCTION void CheckParams() const {
+    // TODO(JMM): I really don't know what to put here...
```

Comment:
I _think_ all `matid` are positive, but this may not _always_ be the case.

Reply:
Yeah I was worried that a user might specify a negative matid for their own custom EOS just to avoid conflicts with the sesame database for example. I'll at least check the min density and temperature are non-negative.